.luacheckrc

```diff
@@ -2,7 +2,7 @@
 allow_defined_top = true
 
 read_globals = {
-	"minetest",
+	"minetest", "core",
```

Comment:
`core` is for Minetest internal code. Please remove this entry regardless on how many warnings there will be.